src/components/CreateExport/CreateExportModal.tsx

```diff
@@ -147,16 +149,17 @@ export function CreateExportModal({
               </Text>
               <DateInput
                 label="Start Date"
-                clearable
+                clearable={false}
                 value={startDate}
                 onChange={setStartDate}
                 {...(endDate && { maxDate: endDate })}
               />
               <DateInput
                 label="End Date"
-                clearable
+                clearable={false}
                 value={endDate}
                 onChange={setEndDate}
+                allowDeselect={false}
                 {...(startDate && { minDate: startDate })}
```

Comment:
Mandatory: Please disallow the deselect for the start date as well.

Optional: Just remove the clearable keyword, as the default is already `false`.